.nvmrc

```diff
@@ -1 +1 @@
-12.19.0
+12.20.0
```

Comment:
I am not sure if we need to update this package as well? 

https://github.com/LiskHQ/lisk-sdk/blob/4409d34102b5bc6703fbc48813bd1f7557ee4a93/package.json#L61

Currently its linked to a year old release.